<!DOCTYPE html>
<!-- saved from url=(0029)http://bootswatch.com/flatly/ -->
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    
    <title>Mohamed Helmy, PhD</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <link rel="stylesheet" href="bootstrap.css" media="screen">
    <link rel="stylesheet" href="custom.min.css">
    <script type="text/javascript" async="" src="ga.js"></script>
   
  <script id="_carbonads_projs" type="text/javascript" src="C6AILKT.json"></script>
  <script type="text/javascript" src="saved_resource"></script>
  </head>
  <body>
     <div class="container">    
    </div>
      <div class="container">
        <div class="row">
          <div class="col-lg-12">
            <div class="bs-component">
              <nav class="navbar navbar-default">
                <div class="container-fluid">
                  <div class="navbar-header">
                    <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                      <span class="sr-only">Toggle navigation</span>
                      <span class="icon-bar"></span>
                      <span class="icon-bar"></span>
                      <span class="icon-bar"></span>
                    </button>
                  </div>
                  <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                   
                    <ul class="nav navbar-nav">
                      <li><a href="index.html">Bio <span class="sr-only">(current)</span></a></li>
                      <li><a href="positions.html">Professional Experiance</a></li>
                      <li class="active"><a href="projects.html">Research</a></li>
                      <li><a href="grants.html">Grants & Recognitions</a></li>
                      <li><a href="publications.html">Publications</a></li>
                      <li><a href="presentations.html">Presentations</a></li>
                    </ul>
                  </div>
                </div>
              </nav>
            </div>

            </div>
          </div>
        </div>
        <div class="container">
      <!-- Example row of columns -->
      <div class="row">
         <div class="col-md-4"><br>
          <img src="hp1.png" width="100%" class="img-rounded img-responsive center-block">
           <h3>Mohamed Helmy, PhD</h3>
           <p>Postdoctoral Researcher, Bader Lab<br> 
           The Donnelly Centre<br>
           University of Toronto</p>
            <h3>Contact Information</h3>
            <p>160 College Street, Room 630 <br>
              The Donnelly Centre<br>
              University of Toronto<br>
              Toronto, Ontario<br>
              Canada, M5S 3E1<br><a href="https://www.google.com/maps?f=q&hl=en&q=160+College+St,+Toronto+ON&om=1">Map</a> <br>
              Email: mohamed.attiashahata[AT]utoronto.ca </p>
    
            <h3>Publication</h3>
            <a href="https://scholar.google.com/citations?user=Ccx_2agAAAAJ&hl=en" target="_blank">Google Scholar Profile</a>
             <br>
			<br>
			<img src="WordCloud2.png" width="90%" class="img-rounded img-responsive left-block">
			<br>
			<br>
			
        </div>
            <div class="col-md-8">
        
        <h3 class="text-success">RSEARCH</h3>
            <p class="text-justify">
            <img src="Reseaerch-Logos.png" class="img-responsive center-block">
            My research interests are highly interdisciplinary, and they lie at the intersections of computational biology/bioinformatics, proteogenomics (proteomics + genomics), systems biology, cancer genomics and network biology. Principally, my research has dealt with <strong>1)</strong> signaling pathway analysis, modeling, and simulations, <strong>2)</strong> Proteogenomics, <strong>3)</strong> Network biology, <strong>4)</strong> software and database systems development.<br>
            <strong>My current  focus is <u>the analysis of disease multi-omics data</u></strong> particularly  assessment of cancer mutations impact on protein interaction networks and gene regulatory networks and finding somatic mutations in disease conditions using proteogenomic approaches. I have a special interest in methods, software and database systems development.
            </p><br>
        <P><strong><h4>Research Interests</h4></strong></P>
        <p class="text-justify"><strong>1. Signaling Pathways Analysis:</strong><br>
        During my internship and my Master's, I worked for two years at the System Immunology Group (Keio University - Japan) in analyzing, modeling and simulating the innate immunity signaling pathways, namely the Toll-like Receptor (TLR)-3, TLR-4 and the Tumor Necrosis Factor (TNF)-α signaling pathways. In this research, I was able to predict several signaling features in the three pathways that were followed later by several experimental confirmations from our group or independent groups. For pathway modeling and simulation I mainly used the E-Cell modeling platform, developed at Keio University, plus several other platforms such as Cellware, Cell designer and CoPaSi.<br>
        Example publications: <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0004661" target="_blank">Helmy <i>et al</i>, PLoS One, 2009</a>, <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0003430" target="_blank">Selvarajoo <i>et al</i>, PLoS One, 2008</a>  
        </p>
            <p><strong>2. Proteogenomics (Proteomics + Genomics):</strong></p>
             <p class="text-justify"><strong>Proteogenomics</strong> is the utilization of large-scale proteome data in genomic analysis e.g. genome annotation. My PhD research focused in developing comprehensive and integrated approach for combining proteomic and genomic data in a single analysis. Particularly, I addressed the problem of the high false discovery rate (FDR) when using 6-frame translated genomic data in the peptide identification process as well as speeding up large-scale and high throughput data analysis in MS-based proteogenomics. I developed the Mass Spectrum Sequential Subtraction (MSSS) method, a novel bioinformatics method that reduces the FDR when searching 6-frame translated genome database and speeds up the peptide sequence identification from large-scale MS-based peptide spectra datasets. MSSS method was later implemented in <a href="http://www.geneffects.com/peppy/">Peppy v2.0</a>, the proteogenomics search tool developed by <a href="http://www.geneffects.com/">Gene Effects Inc. (USA)</a>.<br>
             Example publications: <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2443.2012.01615.x/full" target="_blank">Helmy <i>et al</i>, Genes to Cells, 2011</a><br>
            <strong>Onco-Proteogenomics</strong> is the application of proteogenomics in cancer. The first attempt to apply a proteogenomics approach in cancer was my work published 2010 where I tried to identify mutated peptides in cancer sample (HeLa cells) using MS/MS proteomics and RNAseq data. I coined the term <i>Onco-Proteogenomics</i> to name this type of analysis and the field was latter named after this term. <br>   
            Example publications: <a href="http://download.springer.com/static/pdf/840/art%253A10.1186%252Fgb-2010-11-s1-p17.pdf?originUrl=http%3A%2F%2Fgenomebiology.biomedcentral.com%2Farticle%2F10.1186%2Fgb-2010-11-s1-p17&token2=exp=1463110594~acl=%2Fstatic%2Fpdf%2F840%2Fart%25253A10.1186%25252Fgb-2010-11-s1-p17.pdf*~hmac=9c84fe50b037fcc890a1f219f2ce6738d3f0ece96184b9b9adf2c44aa50e3137" target="_blank">Helmy <i>et al</i>, Genome Biology, 2010</a><br>
            Example of publications acknowledged me coining the term Onco-Proteogenomics:
            <a href="http://www.nature.com/nmeth/journal/v11/n11/abs/nmeth.3138.html" target="_blank">Alfaro <i>et al</i>, Nature Methods, 2014</a>,
            <a href="http://www.tandfonline.com/doi/abs/10.1586/14789450.2016.1132169" target="_blank">Locard-Paulet <i>et al</i>, Expert Review of Proteomics, 2016</a>, 
            <a href="http://www.tandfonline.com/doi/abs/10.1586/14789450.2016.1136217" target="_blank">Subbannayya <i>et al</i>, Expert Review of Proteomics, 2016</a><br></p>
            <p><strong>3. Network Biology:</strong></p>
            <p class="text-justify">My current focus is understanding the role of mutations in cancer and how they affect the protein interaction networks and the gene regulatory networks. I am developing new methods to computationally predict the mutations that affects the binding peptides in the proteins and the binding sites in the gene promoters. Through identifying these mutations, we can predict the cancer driver genes and experimentally validate the best predictions. Such methods are very important now with the rapid accumulation of cancer mutations (41 million mutations to date – <a href="https://dcc.icgc.org/" target="">ICGC data portal</a>). This work is part of the <a href="https://icgc.org/" target="_blank">International Cancer Genome Consortium (ICGC)</a>.
            </p>
        
            <p><strong>4. Methods, Software Tools and Online Resources Development:</strong></p>
            <p class="text-justify"><strong>The Mass Spectrum Sequentail Subtraction Method (MSSS)</strong>: In mass spectrometry-based proteomics, six-frame genome searches present a major since the vast majority of peptides populating six-frame databases do not exist in nature, which increases the chances for false positive spectrum matches. This lead to spurious inflation in the number of “novel” identifications. The MSSS method addresses this point when searching multiple databases. The process involves searching proteomic and genomic databases in sequence with protein database searches preceding the genomic. For each database searched, the 1% FDR was estimated and all spectra identified at that cutoff were removed from the body of spectra used in the next database search. Removing positively identified spectra with each database makes the searchers 50% faster searches and neatly addresses the false positive conundrum for the novel, genome-only matches.<br>
            Example publications: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22686349" target="_blank">Helmy <i>et al</i>, Genes to Cells, 2012</a>     
            </p>
            <p class="text-justify"><strong>The ProteoGenomics Features Evaluator (PGFeval)</strong>: PGFeval is a software tool designed and developed for the evaluation of the proteogenomic novelty of the peptides identified using means of high throughput proteomic approach. Further, PGFeval visualize the gene annotation including the identified peptides, its positions and its proteogenomic novelty.<br>
            Example publications: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21486466" target="_blank">Helmy <i>et al</i>, BMC Plant Biology, 2011</a>          
            </p>
            <p class="text-justify"><strong>The Rice Proteogenomics Database (OryzaPG-DB)</strong>: OryzaPG-DB is the rice proteome database based on shotgun proteogenomics that contains peptides obtained from shotgun-based proteomics and phosphoproteomics with their product ion spectra, as well as updated annotations, side by side with the corresponding protein, cDNA, transcript and genomic sequences and information.<br>
            Example publications: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22639657" target="_blank">Helmy <i>et al</i>, Frontiers of Plant Sciences, 2012</a>, 
            <a href="http://oryzapg.iab.keio.ac.jp/" target="_blank">URL</a>        
            </p>
            <p class="text-justify"><strong>The Cancer Variants Database (Can-VD)</strong>: The Cancer Variants Database (Can-VD) is an online resource for the assessment of the impact of cancer mutations on protein-protein interactions (PPI) networks. Can-VD stores the PPI interaction networks mediated by wildtype and cancer variants and visualizes the overlay of the two networks in order to understand the effects of mutations on the network, and consequently, their cellular and biological impact.<br>
            <span class="text-success">Under preparation</span>, <a href="http://canvd.baderlab.org/" target="_blank">URL</a>   
            </p>
            <p class="text-justify"><strong>The Interface-based Protein Interaction Networks Database (IBPN-DB)</strong>: The Interface-based Protein Interaction Networks Database (IBPN-DB) is an online resource for gathering and storing the published interface-based (IB) protein-protein interaction networks (IN) predicted using any PPI prediction method. TOR-IBIN aims to make these networks available in a central repository with unified format and comprehensive search functions that maximize the use of these data. .<br>
            <span class="text-success">Under preparation</span>
            </p>
       </div>
      </div>
           
        </div>
    <script src="jquery-1.10.2.min.js"></script>
    <script src="bootstrap.min.js"></script>
    <script src="custom.js"></script>
    </body>
</html>