<!DOCTYPE html>
<!-- saved from url=(0029)http://bootswatch.com/flatly/ -->
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    
    <title>Mohamed Helmy, PhD</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <link rel="stylesheet" href="bootstrap.css" media="screen">
    <link rel="stylesheet" href="custom.min.css">
    <script type="text/javascript" async="" src="ga.js"></script>
   
  <script id="_carbonads_projs" type="text/javascript" src="C6AILKT.json"></script>
  <script type="text/javascript" src="saved_resource"></script>
  </head>
  <body>
     <div class="container">    
    </div>
      <div class="container">
        <div class="row">
          <div class="col-lg-12">
            <div class="bs-component">
              <nav class="navbar navbar-default">
                <div class="container-fluid">
                  <div class="navbar-header">
                    <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                      <span class="sr-only">Toggle navigation</span>
                      <span class="icon-bar"></span>
                      <span class="icon-bar"></span>
                      <span class="icon-bar"></span>
                    </button>
                  </div>
                  <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                   
                    <ul class="nav navbar-nav">
                      <li><a href="index.html">Bio <span class="sr-only">(current)</span></a></li>
                      <li><a href="positions.html">Professional Experiance</a></li>
                      <li><a href="projects.html">Research</a></li>
                      <li ><a href="grants.html">Grants & Recognitions</a></li>
                      <li class="active"><a href="publications.html">Publications</a></li>
                      <li><a href="presentations.html">Presentations</a></li>
                    </ul>
                  </div>
                </div>
              </nav>
            </div>

            </div>
          </div>
        </div>
        <div class="container">
      <!-- Example row of columns -->
      <div class="row">
         <div class="col-md-4"><br>
          <img src="hp1.png" width="100%" class="img-rounded img-responsive center-block">
           <h3>Mohamed Helmy, PhD</h3>
           <p>Postdoctoral Researcher, Bader Lab<br> 
           The Donnelly Centre<br>
           University of Toronto</p>
            <h3>Contact Information</h3>
            <p>160 College Street, Room 630 <br>
              The Donnelly Centre<br>
              University of Toronto<br>
              Toronto, Ontario<br>
              Canada, M5S 3E1<br><a href="https://www.google.com/maps?f=q&hl=en&q=160+College+St,+Toronto+ON&om=1">Map</a> <br>
              Email: mohamed.attiashahata[AT]utoronto.ca </p>
    
            <h3>Publications</h3>
            <a href="https://scholar.google.com/citations?user=Ccx_2agAAAAJ&hl=en" target="_blank">Google Scholar Profile</a>
             <br>
			<br>
			<img src="WordCloud2.png" width="90%" class="img-rounded img-responsive left-block">
			<br>
			<br>
			
        </div>
       <div class="col-md-8">
        
        
        <h3 class="text-success">PUBLICATIONS</h3>
            <p class="text-justify"><br> * Corresponding author, ** Equal first author
                <strong><h4>A. Journals</h4></strong>
                
                <p class="text-justify">17- <strong>Helmy, M.</strong>, Wagih O., Jain S., Kuang K. Bader G.D. Cancer high-resolution domain-based interaction networks with integrated mutation impacts.<br>
                <span class="text-success">Under preparation</span></p>
                
                <p class="text-justify">16- <strong>Helmy, M.</strong>, Mee M., Bader G.D. openPIP: The Open Source Protein Interactions Platform.<br>
                <span class="text-success">Under preparation</span></p>
                
                <p class="text-justify">15- Shah, M., <strong>Helmy, M.</strong>, Bader G.D. Utilizing biological data in conjunction with supervised machine learning to identify potential regulatory regions and their sizes throughout the human genome.<br>
                <span class="text-success">Under preparation</span></p>
                
                <p class="text-justify">14- <strong>Helmy, M.</strong>, Crits-Christoph A., Wagih O., Bader G.D DV-Impact: An Online Resource and Data Standard for Reporting Disease Variants Impact on Domain-based Protein Interactions.<br>
                <span class="text-success">Under preparation</span></p>
                
                <p class="text-justify">13- Awad, M., Fahmy R.M., Mosa K. A., <strong>Helmy, M.</strong>, El-Feky F. A. Identification of Effective DNA Barcodes for Triticum Plants through Chloroplast Genome-wide Analysis.<br>
                <span class="text-success">In revision</span></p>
                
                <p class="text-justify">12- Tong J., <strong>Helmy, M.</strong><sup>**</sup>., et al,.  Systems level analysis of the protein-phosphotyrosine landscape in acute myeloid leukemia as a product of activated tyrosine kinases and phosphotyrosine phosphatases.<br>
                <span class="text-success">In revision</span></p>
               
                <p class="text-justify">11- <strong>Helmy, M.</strong>, Crits-Christoph A., Bader G.D. (2016) Ten Simple Rules for Developing Biological Database.PLOS Computational Biology, 12(11): e1005128.<br>
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27832061" target="_blank">PubMed Abstract</a> - <a href="http://10.1371/journal.pcbi.1005128" target="_blank">DOI</a> - <a href="Papers/Helmy-etal-2016-PLoSComputationalBiology.pdf" target="_blank">PDF</a></p>
                 
                <p class="text-justify">10- <strong>Helmy, M.</strong>, Awad M., Mosa K. A. (2016) Limited resources of genome sequencing in developing countries: Challenges and solutions. Applied & Translational Genomics, 9,15:19. <br>
                <a href="doi:10.1016/j.atg.2016.03.003" target="_blank">PubMed Abstract</a> - <a href="http://dx.doi.org/10.1016/j.atg.2016.03.003" target="_blank">DOI</a> - <a href="Papers/Helmy-etal-2916-Applied&TranslationalGenomics.pdf" target="_blank">PDF</a></p>
                
                <p class="text-justify">9- Mosa K. A., , Saadoun I., Kumar K., <strong>Helmy, M.</strong>, Dhankher O. P. (2016) Potential Biotechnological Strategies for the Cleanup of Heavy Metals and Metalloids. Frontiers in Plant Science 7, 303.<br>
                <a href="">PubMed Abstract</a> - <a href="http://doi:10.3389/fpls.2016.00303" target="_blank">DOI</a> - <a href="Papers/Musa-etal-2016-FrontiersPlantScience.pdf" target="_blank">PDF</a></p>
                
                <p class="text-justify">8- Awad M., Ouda O., El-Rifi A., El-Fiki F., Mosa K. A., <strong>Helmy, M.</strong><sup>*</sup>., (2015) FN-Identify: Novel Restriction Enzymes-Based Method for Bacterial Identification in Absence of Genome Sequencing. Advances in Bioinformatics Article ID 303605, doi.org/10.1155/2015/303605.<br>
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26880910" target="_blank">PubMed Abstract</a> - <a href="http://dx.doi.org/10.1155/2015/303605" target="_blank">DOI</a> -  <a href="Papers/Awad-etal-2015-AdvancesInBioinformatics.pdf" target="_blank">PDF</a></p>
                
                <p class="text-justify">7- El-Metwally, S, Hamza, T, Zakarya, M and <strong>Helmy, M.</strong><sup>*</sup>. (2013)Next-Generation Sequence Assembly: Four Stages of Data Processing and Computational Challenges, PLOS Computational Biology, 9 (12), e1003345.<br>
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24348224" target="_blank">PubMed Abstract</a> - <a href="http://dx.doi.org/10.1371/journal.pcbi.1003345" target="_blank">DOI</a> - <a href="Papers/El-Metwally-etal-2013-PLoSComputationalBiology.pdf" target="_blank">PDF</a></p>
                 
                <p class="text-justify">6- <strong>Helmy, M.</strong>, Tomita M, Ishihama Y. (2012) Peptide identification by searching large-scale tandem mass spectra against large databases: bioinformatics methods in proteogenomics. Genes, Genomes and Genomics, 6(SI1), 76-85.<br>
                <a href="https://www.researchgate.net/profile/Mohamed_Helmy3/publication/269574159_Peptide_identification_by_searching_large-scale_tandem_mass_spectra_against_large_databases_bioinformatics_methods_in_proteogenomics/links/548f17bf0cf225bf66a7fafd.pdf" target="_blank">ResearchGate</a> - <a href="Papers/Helmy-etal-2012-GeneGenomeGenomics.pdf" target="_blank">PDF</a></p>
                
                <p class="text-justify">5- <strong>Helmy, M.</strong>, Sugiyama N, Tomita M, Ishihama Y. (2012) The rice proteogenomics database OryzaPGDB: development, expansion and new features. Frontiers of Plant Sciences, 3:65, doi:10.3389/fpls.2012.00065.<br>
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/22639657" target="_blank">PubMed Abstract</a> - <a href="http://dx.doi.org/10.3389/fpls.2012.00065" target="_blank">DOI</a> - <a href="Papers/Helmy-etal-2012-PlantSciences.pdf" target="_blank">PDF</a></p>
                
                <p class="text-justify">4- <strong>Helmy, M.</strong>, Tomita, M., Ishihama Y. (2011) OryzaPG-DB: Rice Proteome Database based on Shotgun Proteogenomics, BMC Plant Biology, 11,63, 2011. doi:10.1186/1471-2229-11-63.<br>
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/21486466" target="_blank">PubMed Abstract</a> - <a href="doi:0.1186/1471-2229-11-63" target="_blank">DOI</a> - <a href="Papers/Helmy-etal-2011-BMCPlantBiology.pdf" target="_blank">PDF</a></p>
                
                <p class="text-justify">3- <strong>Helmy, M.</strong>, Sugiyama N, Tomita M, Ishihama Y. (2012) Mass Spectrum Sequential Subtraction: bioinformatics method facilitates searching large dataset of peptide MS/MS spectra against large nucleotide databases for proteogenomics. Genes to Cells, 17(8), 633-644, doi:10.1111/j.1365-2443.2012.01615.<br>
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/22686349" target="_blank">PubMed Abstract</a> - <a href="DOI:10.1111/j.1365-2443.2012.01615.x" target="_blank">DOI</a> - <a href="Papers/Helmy-etal-2012-GenesToCells.pdf" target="_blank">PDF</a></p>
                
                <p class="text-justify">2- <strong>Helmy, M.</strong>., Gohda, J., Inoue, J., Tomita, M., Tsuchiya, M. and Selvarajoo, K. (2009) Predicting novel features of Toll-like Receptor 3 signalling in macrophages, PLoS ONE, 4(3): e4661. doi:10.1371/journal.pone.0004661.<br>
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/19252739" target="_blank">PubMed Abstract</a> - <a href="http://dx.doi.org/10.1371/journal.pone.0004661" target="_blank">DOI</a> - <a href="Papers/Musa-etal-2016-FrontiersPlantScience.pdf" target="_blank">PDF</a></p>
                
                <p class="text-justify">1- Selvarajoo, K., Takabe, Y., Gohda, J., <strong>Helmy, M.</strong>, Akira, S., Tomita, M., Tsuchiya, M., Inoue, J. and Matsuo, K. (2008) Signalling Flux Redistribution at Toll-like Receptor Pathway Junctions., PLoS ONE , 3(10) , e3430, doi:10.1371/journal.pone.0003430.<br>
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/18927610" target="_blank">PubMed Abstract</a> - <a href="http://doi:10.1371/journal.pone.0003430" target="_blank">DOI</a> - <a href="Papers/Selvarajoo-etal-2008-PLoSONE.pdf" target="_blank">PDF</a></p>
                
                
                
               
                </p>
            <br>
            <p>
                <strong><h4>B. Books</h4></strong>
				<p class="text-justify">2- Mosa K. A., El-Shabrawi H., Ismail A., and <strong>Helmy, M.</strong> (In Press) Plant Stress Tolerance: An Integrated Omics Approaches, Springer Briefs in Systems Biology, Springer, PA, USA.</p>

                <p class="text-justify">1- El-Metwally S., Ouda O, and <strong>Helmy, M.</strong><sup>*</sup> (2014) Next Generation Sequencing Challenges in Technologies and Sequence Assembly, Springer Briefs in Systems Biology, (7) ISBN: 978-1-4939-0714-4 (Print) 978-1-4939-0715-1 (Online), Springer, PA, USA. <br>
                <a href="http://www.springer.com/us/book/9781493907144" target="_blank">Springer</a>, <a href="https://www.amazon.ca/Generation-Sequencing-Technologies-Challenges-SpringerBriefs-ebook/dp/B00JT3Q4NQ/ref=sr_1_1?ie=UTF8&qid=1462977552&sr=8-1&keywords=Next+Generation+Sequencing+Technologies+and+Challenges" target="_blank">Amazon</a>, <a href="https://docs.google.com/forms/d/1re8WOHD6EolaY_C8gG6e1ff-U31zezcIxTy4xeoxytU/viewform" target="_blank">Request free copy</a>  </p>
                
</p>
            <br>
            <p>
                <strong><h4>C. Proceedings</h4></strong>
                <p class="text-justify">2- <strong>Helmy, M.</strong>, Tomita M., Tsuchiya M., Selvarajoo K. (2008) In silico analysis of Toll-Like receptor 3 pathways. Proceedings of Bioinformatica Indica'08. India 2008.<br>
                <a href="https://www.researchgate.net/publication/261879170_In_silico_analysis_of_Toll-Like_receptor_3_pathways" target="_blank">ResearchGate</a></p>
                
                <p class="text-justify">1- Selvarajoo K., <strong>Helmy, M.</strong>, Tomita M., Tsuchiya M. (2008) Inferring the mechanistic basis for the dynamic response of the MyD88-dependent and –independent pathways. 10 on Molecular Systems Biology 2008 (ICMSB'08).Philippine 2008.<br><a href="https://www.researchgate.net/publication/230823766_Inferring_the_mechanistic_basis_for_the_dynamic_response_of_the_MyD88-dependent_and_-independent_pathways" target="_blank">ResearchGate</a></p>
                
                
                
            </p>
            <br>
            <p>
                <strong><h4>D. Thesis</h4></strong>
                <p class="text-justify">2- PhD thesis: <strong>Helmy, M.</strong>  (2012) Systems biology approaches to improve genome annotations through development of bioinformatics pipeline for large-scale mass spectrometry-based proteogenomics. Supervisors: Professor Masaru Tomita, Peofessor Yasushi Isgihama, Professor Akio Kanai, Keio University, Tokyo, Japan. <br>
                <a href="Thesis/PhD-Thesis.pdf" target="_blank">full Text PDF</a></p>
                
                <p class="text-justify">1- Master's thesis: <strong>Helmy, M.</strong> (2009) Systems biology research on: 1) Computational analysis of Toll-like receptor 3 signaling pathway, and 2) Exploiting proteogenomics for improving Oryzae sativa genome annotation. Supervisors: Professor Masaru Tomita, Peofessor Yasushi Isgihama, Professor Kumar Selvarajoo, Keio University, Tokyo, Japan. <br>
                <a href="Thesis/Masters-Thesis.pdf" target="_blank">Full Text PDF</a></p>
                
                
                
                
            </p>
          
       </div>
      </div>
           
        </div>
            
      </div>
           
        </div>
    <script src="jquery-1.10.2.min.js"></script>
    <script src="bootstrap.min.js"></script>
    <script src="custom.js"></script>
    </body>
</html>